snippets/11_others.txt

```diff
@@ -1,4 +1,4 @@
-## Apple 苹果
+﻿## Apple 苹果

```

Comment:
It seems that there is a BOM (U+FEFF), we don't have this in other files however, could you please remove it?